clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -8,6 +8,13 @@
 from pandas import DataFrame
 
 
+def check_clinical_path(path_to_clinical_data: PathLike) -> None:
```

Comment:
Might be a bit overkill to have a function which only checks whether the input is None or not. WDYT ?

Reply:
Yes it is aha. I'll change it